src/wp-includes/rest-api/endpoints/class-wp-rest-template-autosaves-controller.php

```diff
@@ -0,0 +1,276 @@
+<?php
+/**
+ * REST API: WP_REST_Template_Autosaves_Controller class.
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 6.4.0
+ */
+
+/**
+ * Core class used to access autosaves via the REST API.
```

Comment:
Two notes.

Copypasta, need to differentiate from underlying controller:
```suggestion
 * Core class used to access template autosaves via the REST API.
```

Second, not sure what our policy is around further elaboration in class documentation is, but should we explain why we need to subclass here, rather than simply leaving the @see link and leaving it up to blame / the reader's interpretation?